tasks/main.yml

```diff
@@ -2,20 +2,20 @@
 # tasks file for ansible-role-docker
 
 - name: Incule variables
-  include_vars: "{{ ansible_os_family }}.yml"
+  include_vars: '{{ ansible_os_family }}.yml'
 
 - name: Prepare system
   include_tasks: prepare-gitlab-runner.yml
 
 - name: adding selinux support
   include_tasks: selinux-support.yml
-  when: ansible_os_family == "RedHat"
+  when: ansible_os_family == 'RedHat'
```

Comment:
Looks like code duplication since we already have 
```yaml
include_vars: '{{ ansible_os_family }}.yml'
```

Reply:
Fixed